iot-e2e-tests/common/src/test/resources/log4j.properties

```diff
@@ -5,7 +5,7 @@ log4j.rootLogger=ERROR, stdout
 log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.Target=System.out
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
-log4j.logger.tests.integration.com.microsoft.azure.sdk.* = DEBUG
```

Comment:
Removing the star here allows for the expected recursion to pick up logs from this package and subpackages